tasklist/client/src/modules/utils/formatDateRelative.ts

```diff
@@ -198,19 +220,15 @@ function formatISODate(dateString: string | null) {
 
 function formatDateTime(time: Date, now?: Date) {
   const resolution = getResolution(time, now ?? new Date());
-  const text = format(time, getFormat(resolution, TEXT_DATETIME_LANG_EN), {
-    locale: enUS,
+  const text = format(time, getFormat(resolution, false, true), {
```

Comment:
Let's use an object as argument so it's easier to understand

```suggestion
  const text = format(time, getFormat({resolution, isSpeech: false, includeTime: true}), {
```